helper/cert/source_gen.go

```diff
@@ -143,9 +150,19 @@ func (s *GenSource) Certificate(ctx context.Context, last *Bundle) (Bundle, erro
 		result.Key = []byte(key)
```

Comment:
Not new in this PR, but this block should probably de-indent out of the `if err == nil` braces, and instead return early `if err != nil`.

Reply:
Done in 51f676243f146b93c2836e9905f09ff69e15a00b.